eng/pipelines/templates/steps/analyze_dependency.yml

```diff
@@ -23,9 +23,7 @@ steps:
     displayName: 'Populate Reports Staging Folder'
     condition: succeededOrFailed()
 
-  - task: PythonScript@0
+  - pwsh: |
+      sdk_analyze_deps --verbose --out "$(Build.ArtifactStagingDirectory)/reports/dependencies.html" --dump "$(Build.ArtifactStagingDirectory)/reports"
     displayName: 'Analyze dependencies'
-    condition: and(succeededOrFailed(),ne(variables['Skip.AnalyzeDependencies'],'true'))
-    inputs:
-     scriptPath: 'scripts/analyze_deps.py'
-     arguments: '--verbose --out "$(Build.ArtifactStagingDirectory)/reports/dependencies.html" --dump "$(Build.ArtifactStagingDirectory)/reports"'
\ No newline at end of file
+    condition: succeededOrFailed()
```

Comment:
Just to be sure, is `variables['Skip.AnalyzeDependencies']` no longer something we're checking for?

Reply:
I'm not certain why that got cleaned out. I should definitely keep this skip around. Thanks for the heads up mccoy!